src/main/resources/init_db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE 'books_db';
+
+CREATE TABLE 'books' (
+    'id' BIGINT PRIMARY KEY NOT NULL AUTO_INCREMENT,
```

Comment:
```suggestion
    'id' BIGINT PRIMARY KEY AUTO_INCREMENT,
```

MySQL automatically add not null and unique when you declare id as PK